test/known_issues/test-fs-fchmod-12835.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+const common = require('../common');
+
+// Windows only: Test fchmod on files with `attrib -a -h -i -r -s`
+// setting RO then can't set RW back
+
+const {strictEqual, ok} = require('assert');
+const {execSync} = require('child_process');
+const fs = require('fs');
+
+ok(common.isWindows, 'reset of test only for windows');
```

Comment:
Since this is a `known-issue` it's supposed to fail.
So it's either this line or exclude this test for all platforms but Windows in `known_issues.status`

Otherwise we get:
```
refael@refaelux:/mnt/d/code/node-cur$ python tools/test.py known_issues
=== release test-fs-fchmod-12835 [negative] ===
Path: known_issues/test-fs-fchmod-12835
1..0 # Skipped: undefined
Command: out/Release/node /mnt/d/code/node-cur/test/known_issues/test-fs-fchmod-12835.js
[00:11|% 100|+  13|-   1]: Done
```